draft-js-mention-plugin/src/mentionSuggestionsStrategy.js

```diff
@@ -4,8 +4,8 @@ import findWithRegex from 'find-with-regex';
 import escapeRegExp from 'lodash.escaperegexp';
 
 export default (trigger: string, regExp: string) => (contentBlock: Object, callback: Function) => {
-  const reg = new RegExp(String.raw({
-    raw: `(\\s|^)${escapeRegExp(trigger)}${regExp}` // eslint-disable-line no-useless-escape
-  }), 'g');
-  findWithRegex(reg, contentBlock, callback);
+  findWithRegex(new RegExp(`(\\s|^)${escapeRegExp(trigger)}${regExp}`, 'g'), contentBlock, (start, end) => {
+    const isAlreadyEntity = contentBlock.getEntityAt(Math.max(end - 1, 0)) !== null;
+    if (!isAlreadyEntity) callback(start, end);
+  });
 };
```

Comment:
Compared to the original file [here](https://github.com/pioul/draft-js-plugins/commit/1bcf2e0a5aea3dc629aabf7f915c58f0f6848155), I'm not sure what the new syntax with `String.raw()` brings to the mix compared to the template literal, but just to be safe we should probably make as little changes to draft-js-plugins's updated source code.

In [that commit](https://github.com/pioul/draft-js-plugins/commit/1bcf2e0a5aea3dc629aabf7f915c58f0f6848155) the only change that was made was adding a custom callback method (the last argument to `findWithRegex()` in that same commit). We could easily pull this change out and integrate it in mentionSuggestionsStrategy.js without changing anything else :)